libldbrest/endpoints.go

```diff
@@ -104,8 +106,8 @@ func getItems(w http.ResponseWriter, r *http.Request, p httprouter.Params) {
 		}
 	}
 
-	w.Header().Set("Content-Type", "application/json")
-	json.NewEncoder(w).Encode(multiResponse{nil, results})
+	w.Header().Set("Content-Type", "application/msgpack")
```

Comment:
I set content-type to `"application/x-msgpack"` in the client. Found that [here](http://stackoverflow.com/questions/6888900/mime-type-for-msgpack)

One of the answers does specify `"application/msgpack"` as the right type. Something about `"x-"` prefixes not being preferred.

I'm ok either way. Should I make the change client-side or do you want to change it here?